v0/destinations/appcues/transform.js

```diff
@@ -0,0 +1,184 @@
+const get = require("get-value");
+const set = require("set-value");
+const {
+  EventType
+} = require("../../../constants");
+
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  getValueFromMessage,
+  removeUndefinedAndNullValues,
+  removeUndefinedAndNullAndEmptyValues,
+} = require("../../util");
+
+const {
+  ConfigCategory,
+  mappingConfig,
+  getEndpoint,
+} = require("./config");
+
+const {
+  utcTimeToEpoch,
```

Comment:
Move it to `util/index.js`. Also, please check if we already have this function there.

Reply:
Yes, we are having a function called toUnixTimeStamp. Now I am using it directly and deleted the function written by me